src/Core/Addon/Module/ModuleManagerBuilder.php

```diff
@@ -171,27 +171,29 @@ private function __construct(bool $isDebug = _PS_MODE_DEV_)
         $prestashopAddonsConfig =
             $yamlParser->parse($this->getConfigDir() . '/addons/categories.yml');
 
-        $clientConfig = $config['csa_guzzle']['clients']['addons_api']['config'];
+        $tools = new Tools();
+        $tools->refreshCaCertFile();
+
+        $clientConfig = $config['eight_points_guzzle']['clients']['addons_api'];
+        $clientConfig['verify'] = _PS_CACHE_CA_CERT_FILE_;
+        if (file_exists($this->getConfigDir() . '/parameters.php')) {
+            $parameters = require $this->getConfigDir() . '/parameters.php';
```

Comment:
Would it be possible to improve how the parameters are retrieved?

Reply:
I don't think it's in the scope of the PR. Isn't it ?